pages/common/mixxx.md

```diff
@@ -0,0 +1,28 @@
+# mixxx
+
+> Free and open source cross-platform DJ software <https://www.mixxx.org>.
+> Command-line arguments are documented at <https://mixxx.org/manual/latest/chapters/appendix.html#command-line-options>.
+
+- Start the Mixxx GUI in full-screen:
+
+`mixxx --fullScreen`
+
+- Start in safe developer mode to debug a crash:
+
+`mixxx --developer --safeMode`
+
+- Debug a malfunction:
+
+`mixxx --debugAssertBreak --developer --loglevel trace`
+
+- Start mixxx using a contained configuration:
+
+`mixxx --resourcePath {{mixxx/res/controllers}} --settingsPath {{settingsPath}}`
```

Comment:
why `controllers`?
```suggestion
`mixxx --resourcePath {{path/to/directory}} --settingsPath {{path/to/directory}}`
```

Reply:
Because that is the canonical location, the flag is usually used in development

Reply:
I see. If it's a domain-specific thing, maybe we can leave that one as-is. However, I'd still suggest changing `settingsPath` to be more consistent:
```suggestion
`mixxx --resourcePath {{mixxx/res/controllers}} --settingsPath {{path/to/file}}`
```